x-pack/solutions/search/plugins/enterprise_search/public/plugin.ts

```diff
@@ -117,13 +113,6 @@ const contentLinks: AppDeepLink[] = [
       defaultMessage: 'Connectors',
     }),
   },
-  {
-    id: 'searchIndices',
```

Comment:
Since we're removing this here we will need to also remove this from the deep links type definitions for `ContentLinkId` [here](https://github.com/elastic/kibana/blob/115dcb3411f0fe65d460773c17e6431208bad950/src/platform/packages/shared/deeplinks/search/deep_links.ts#L49)